.github/workflows/everything.yml

```diff
@@ -145,7 +145,11 @@ jobs:
 
   linux_emu:
     needs: [format, git_checks]
-    if: false # needs.git_checks.outputs.num_code_changes > 0
+    if: needs.git_checks.outputs.num_code_changes > 0 &&
```

Comment:
@caitlinross this is the only diff. It means just to run in master/release* and when code changes